tests/test-helper.js

```diff
@@ -1,17 +1,13 @@
 import RSVP from 'rsvp';
-
-import Application from '@ember/application';
 import resolver from './helpers/resolver';
 import { setResolver } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
 
 import QUnit from 'qunit';
 import DS from 'ember-data';
 import { wait, asyncEqual, invokeAsync } from 'dummy/tests/helpers/async';
-import loadInitializers from 'ember-load-initializers';
 
 setResolver(resolver);
```

Comment:
We should change this to setApplication (not blocking)

Reply:
Got a link handy for what that looks like?

Reply:
This is what a new addons looks like: https://github.com/ember-cli/ember-addon-output/blob/master/tests/test-helper.js

Reply:
Ok, looked into this and it appears that until we've killed the use of `createStore` that this seems to cause us too many issues (specifically we have tests without containers and tests for which this causes us to create multiple application instances). I will leave a TODO with this note.

Reply:
Was able to get us further and moved the resolver setup into `createStore`'s domain. We have ~1100 of 1500 tests still setting up the store using this helper, so the burndown on it is likely to take a while.